### PR TITLE
CI Fix

### PR DESCRIPTION
**Pull request by Imipenem**

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -61,23 +66,22 @@ jobs:
               uses: actions/checkout@v2.3.4
 
             - name: Set up Python ${{ matrix.python-version }}
-              uses: actions/setup-python@v2.2.2
+              uses: actions/setup-python@v2.1.4
```
Comment (Zethson) [inline .github/workflows/run_tests.yml:69]: Why?

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -103,18 +107,19 @@ jobs:
                       ${{ steps.pre-commit-cache.outputs.result }}-
 
             - name: Run Nox
-              run: nox --force-color --python=${{ matrix.python-version }}
+              run: |
```
Comment (Zethson) [inline .github/workflows/run_tests.yml:110]: In ehrapy I've decided to put single line commands without a `|` and only if it's multiple lines to include the `|`. What do you think?

File: .github/workflows/run_tests.yml [hunk 3]
```diff
@@ -103,18 +107,19 @@ jobs:
...
+                  nox --force-color --python=${{ matrix.python-version }}
 
             - name: Upload coverage data
               if: always() && matrix.session == 'tests'
-              uses: "actions/upload-artifact@v2.2.4"
+              uses: "actions/upload-artifact@v2.2.3"
```
Comment (Zethson) [inline .github/workflows/run_tests.yml:115]: Why?

File: .github/workflows/run_tests.yml [hunk 4]
```diff
@@ -103,18 +107,19 @@ jobs:
...
               with:
                   name: coverage-data
                   path: ".coverage.*"
 
             - name: Upload documentation
               if: matrix.session == 'docs-build'
-              uses: actions/upload-artifact@v2.2.4
+              uses: actions/upload-artifact@v2.2.3
```
Comment (Zethson) [inline .github/workflows/run_tests.yml:122]: ?

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/workflows/run_tests.yml [hunk 5]
```diff
@@ -162,4 +160,4 @@ jobs:
               run: nox --force-color --session=coverage -- xml -i
 
             - name: Upload coverage report
-              uses: codecov/codecov-action@v1.5.2{% endraw %}
+              uses: codecov/codecov-action@v1.5.2
```
Comment (Zethson) [inline cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/workflows/run_tests.yml:163]: Why?

File: .github/workflows/run_tests.yml (hunk 2 above)
Comment (Imipenem) [reply .github/workflows/run_tests.yml:105]: Hm, guess for single lines, I'd prefer the version without `|` just like in ehrapy

**Review by Zethson — APPROVED** [review]
LGTM

**Pull request merged**